stream_chat/client.py

```diff
@@ -598,3 +605,17 @@ def delete_push_provider(self, provider_type: str, name: str) -> StreamResponse:
 
     def list_push_providers(self) -> StreamResponse:
         return self.get("push_providers")
+
+    def create_import_url(self, filename: str) -> StreamResponse:
+        return self.post("import_urls", data={"filename": filename})
```

Comment:
maybe it makes sense to wrap this into a helper that does the uploading too, what do you think @ferhatelmas ?

Reply:
we dont have it in other (js, go) sdk's. I think it's better to leave it to the customer

Reply:
consistent might be better

Reply:
Is also saw that Peter included an example in the doc string. LGTM 🙌